### PR TITLE
Transferring code snippets for igRating

### PR DESCRIPTION
**Pull request by Marina-L-Stoyanova**
#295

### CODE REVIEW

File: src/js/modules/infragistics.ui.rating.js [hunk 1]
```diff
@@ -47,57 +47,236 @@
...
+					vertical : true
+				});
+
+				//Get
+				var isVertical = $(".selector").igRating("option", "vertical");
+				```	
+			*/
 			vertical: false,
-			/* type="number" Gets or sets value (selected votes or percent).  */
+			/* type="number" Gets or sets value (selected votes or percent).
```
Comment (kdinev) [inline src/js/modules/infragistics.ui.rating.js]: So, can this option also be a string, if percent can be set?

File: src/js/modules/infragistics.ui.rating.js [hunk 2]
```diff
@@ -47,57 +47,236 @@
...
+				//Get
+				var value = $(".selector").igRating("option", "value");
+
+				//Set
+				$(".selector").igRating("option", "value", 3);
+				```
+			*/
 			value: null,
-			/* type="number" Gets or sets value-hover (hovered votes or percent of hovered votes). Default is same as value. */
+			/* type="number" Gets or sets value-hover (hovered votes or percent of hovered votes). Default is same as value.
```
Comment (kdinev) [inline src/js/modules/infragistics.ui.rating.js]: Same question as above.

File: src/js/modules/infragistics.ui.rating.js [hunk 3]
```diff
@@ -47,57 +47,236 @@
...
 			precisionZeroVote: 0.25,
 			/* type="number" Gets or sets number of decimal places used to round value and value-hover.
 				Negative value will disable that option and value will not be rounded.
 				Notes:
 				If precision is "whole" or "half" and roundedDecimalPlaces is set in range of 0..2, then 3 is used.
 				If valueAsPercent is enabled and roundedDecimalPlaces is set to 0, then 1 is used.
 				If it is larger than 15, then 15 is used.
-				*/
+				```
+				 //Initialize
```
Comment (kdinev) [inline src/js/modules/infragistics.ui.rating.js]: There is one additional space in the indentation.

File: src/js/modules/infragistics.ui.rating.js [hunk 4]
```diff
@@ -47,57 +47,236 @@
...
+				.redmond .ui-igrating-vote { ... }
+				.redmond .ui-igrating-voteselected { ... }
+				.redmond .ui-igrating-votehover { ... }
+				.redmond .ui-igrating-votedisabled { ... }
+				.redmond .ui-igrating-votedisabledselected { ... }
+				```
+			*/
 			theme: null,
 			/* type="object" Gets or sets object which contains options supported by igValidator.
 				Note that for onblur validation depends on the "focusable" option.
```
Comment (kdinev) [inline src/js/modules/infragistics.ui.rating.js]: Make a link to the focusable option and remove the quotes.

File: src/js/modules/infragistics.ui.rating.js [hunk 5]
```diff
@@ -109,7 +288,38 @@
 				{ 1: { 0: "normalCss", 1: "selectedCss", 2: "hoverCss"} }
 				will customize only second vote with normalCss for normal state, hoverCss for hover state and selectedCss for selected state.
```
Comment (kdinev) [inline src/js/modules/infragistics.ui.rating.js]: Again, link to the css classes.

File: src/js/modules/infragistics.ui.rating.js [hunk 6]
```diff
@@ -462,6 +707,13 @@
 		validator: function (destroy) {
 
 			/* Gets reference to igValidator used by igRating.
```
Comment (kdinev) [inline src/js/modules/infragistics.ui.rating.js]: Link to the validator API.

**Comment by Marina-L-Stoyanova** [comment]
I believe I fixed all mentioned points. If you notice anything else, just let me know.

**Comment by kdinev** [comment]
@Marina-L-Stoyanova Looks good! Once the PR build passes, I will merge.

**Pull request merged**